src/pages/JoinComplete/JoinComplete.DiscordNotice.tsx

```
@@ -0,0 +1,34 @@
+    <div className="break-words pt-8">
+      <p className="line-breaks mb-1 text-center text-lg">
+        앞으로의 모든 활동과 소통은 <strong>디스코드</strong>에서 이루어집니다.
+        Aegis와 함께 성장해 나가요!
+      </p>
+      <Button
+        size="lg"
+        className="w-full bg-discord-blue text-white hover:bg-discord-blue active:scale-95 transition-transform duration-100"
+        asChild
+      >
+        <a
+          href={import.meta.env.VITE_DISCORD_NOTICE_URL}
```

Biome (1.9.4)

[error] These CSS classes should be sorted.

Safe fix: Sort the classes.


(lint/nursery/useSortedClasses)